scripts/publish.sh

```diff
@@ -122,6 +122,17 @@ publish_to_docker_hub() {
 		if [ "$IMAGE_SHA_MATCHED" = "FALSE" ]; then
 			create_manifest_list ${1} "stable" ${AWS_FOR_FLUENT_BIT_STABLE_VERSION}
 		fi
+
+		docker pull ${1}:"$init"-stable || echo "0"
```

Comment:
hmmm  either I didn't notice this before or I no longer remember why its `:"$init"-stable` shouldn't it just be `init-stable`...

Reply:
Hmm, let me recall...

I think I did this because we need to put different suffixes after "init" other than "stable".
Like [here](https://github.com/aws/aws-for-fluent-bit/pull/395/files#diff-c63759f58a022cf694046a961470c6e28c042d8f221646bc5ddbb5fa959dda75R127) and [here](https://github.com/aws/aws-for-fluent-bit/pull/395/files#diff-c63759f58a022cf694046a961470c6e28c042d8f221646bc5ddbb5fa959dda75R247).

And sometimes we need to use "init stable" rather than "init-stable" like [here](https://github.com/aws/aws-for-fluent-bit/pull/395/files#diff-c63759f58a022cf694046a961470c6e28c042d8f221646bc5ddbb5fa959dda75R224).

Due to the different code styles and parameter passing way in this file so I chose not to hard code "init".

Reply:
Thank you @Galaoaoa for showing strong ownership and continuing to track this PR :) 

> I think I did this because we need to put different suffixes after "init" other than "stable".

Hmm but I don't understand why we need the `"$init"`... the `init` part never changes right? Its the bit afterwards. So I'd expect to see: `init-${var}`.

Reply:
Yeah, we can use init instead of "$init". From the current perspective, I think we could indeed just use the init, this would be the better way :)